mass_mailing_unsubscribe_af/i18n/sv_SE.po

```diff
@@ -254,45 +256,45 @@ msgstr ""
 #: code:addons/mass_mailing_unsubscribe_af/static/src/js/unsubscribe.js:254
 #, python-format
 msgid "You are not authorized to do this!"
-msgstr ""
+msgstr "Du är inte behörig att göra detta!"
 
 #. module: mass_mailing_unsubscribe_af
 #. openerp-web
 #: code:addons/mass_mailing_unsubscribe_af/static/src/js/unsubscribe.js:105
 #, python-format
 msgid "You have been <strong>successfully unsubscribed from %s</strong>."
-msgstr ""
+msgstr "Du har <strong>avslutat prenumerationen från %s</strong>."
 
 #. module: mass_mailing_unsubscribe_af
 #. openerp-web
 #: code:addons/mass_mailing_unsubscribe_af/static/src/js/unsubscribe.js:110
 #, python-format
 msgid "You have been <strong>successfully unsubscribed</strong>."
-msgstr ""
+msgstr "Du har <strong>avslutat prenumerationen</strong>."
 
 #. module: mass_mailing_unsubscribe_af
 #. openerp-web
 #: code:addons/mass_mailing_unsubscribe_af/static/src/js/unsubscribe.js:210
 #, python-format
 msgid "You have been successfully <strong>added to our blacklist</strong>. You will not be contacted anymore by our services."
-msgstr ""
+msgstr "Du har <strong>lagts till vår svartlista</strong>. Du kommer inte att kontaktas längre av våra tjänster."
```

Comment:
Missing "i"
I believe that its more correct: "Du har <strong lagts till i vår...